src/everest/detached/jobs/everserver.py

```diff
@@ -273,7 +274,8 @@ def get_shared_data(
 
 
 def _find_open_port(host, lower, upper) -> int:
-    for port in range(lower, upper):
+    for _ in range(5):
```

Comment:
Just 5 tries seem a low number.

Reply:
It tries 5 times to randomly find a port not in use in the range 5000 to 5800. Should be low probability of not finding one